notebook/notebook-worker-images

```diff
@@ -0,0 +1 @@
+gcr.io/hail-vdc/hail-jupyter:e3f9a751f0a837815afeaf6fff8057f04747a35c908fb1ddf7cad6ad5cd428cd
```

Comment:
Ok, happy to do this, and at the same time think it would be better for a 2nd PR. Right now if we accept the PR without this change, app.hail.is/notebook will begin working again. Right now it is broken.

If you're firm on this, I'll start on it tomorrow.